tests/integ/datasets.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import gzip
+import os
+import pickle
+
+from tests.integ import DATA_DIR
+
+
+def one_p_mnist():
```

Comment:
I thought about it, but wasn't convinced that it deserved to be a top-level fixture (though I could very well be wrong about that), and defining fixtures that get imported looks a bit messy because you have to throw in some `#noqa`s.

maybe fixtures for all our datasets is the way to go, though. I'll admit I haven't put much thought into it.